### ISSUE TITLE
Thread Blocked on Startup

### ISSUE THREAD

**Issue by dajenacuko** [gentics/mesh#996]
## Gentics Mesh Version, operating system, or hardware.
- v1.3.2

## Operating System
- MacOSX (Docker Envinronment)

## Problem
Thread will be blocked on Gentics Mesh Startup.  Here are the logs:

## Logs 
```
 13:31:19.072 [] WARN  [vertx-blocked-thread-checker] - Thread Thread[vert.x-eventloop-thread-2,5,main]=Thread[vert.x-eventloop-thread-2,5,main] has been blocked for 11076 ms, time limit is 2000 ms
mesh_1           | io.vertx.core.VertxException: Thread blocked
mesh_1           | 	at app//com.fasterxml.jackson.databind.ser.SerializerCache.untypedValueSerializer(SerializerCache.java:92)
mesh_1           | 	at app//com.fasterxml.jackson.databind.SerializerProvider.findValueSerializer(SerializerProvider.java:549)
mesh_1           | 	at app//com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.acceptJsonFormatVisitor(DefaultSerializerProvider.java:566)
mesh_1           | 	at app//com.fasterxml.jackson.databind.ObjectWriter.acceptJsonFormatVisitor(ObjectWriter.java:1051)
mesh_1           | 	at app//com.fasterxml.jackson.databind.ObjectWriter.acceptJsonFormatVisitor(ObjectWriter.java:1058)
mesh_1           | 	at app//com.fasterxml.jackson.module.jsonSchema.JsonSchemaGenerator.generateSchema(JsonSchemaGenerator.java:89)
mesh_1           | 	at app//com.gentics.mesh.json.JsonUtil.getJsonSchema(JsonUtil.java:226)
mesh_1           | 	at app//com.gentics.mesh.rest.impl.InternalEndpointRouteImpl.exampleResponse(InternalEndpointRouteImpl.java:321)
mesh_1           | 	at app//com.gentics.mesh.core.endpoint.user.UserEndpoint.addUpdateHandler(UserEndpoint.java:204)
mesh_1           | 	at app//com.gentics.mesh.core.endpoint.user.UserEndpoint.registerEndPoints(UserEndpoint.java:54)
mesh_1           | 	at app//com.gentics.mesh.rest.RestAPIVerticle.registerEndPoints(RestAPIVerticle.java:286)
mesh_1           | 	at app//com.gentics.mesh.rest.RestAPIVerticle.start(RestAPIVerticle.java:204)
mesh_1           | 	at app//io.vertx.core.impl.DeploymentManager.lambda$doDeploy$9(DeploymentManager.java:556)
mesh_1           | 	at app//io.vertx.core.impl.DeploymentManager$$Lambda$416/0x0000000100637840.handle(Unknown Source)
mesh_1           | 	at app//io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:369)
mesh_1           | 	at app//io.vertx.core.impl.EventLoopContext.lambda$executeAsync$0(EventLoopContext.java:38)
mesh_1           | 	at app//io.vertx.core.impl.EventLoopContext$$Lambda$417/0x0000000100637440.run(Unknown Source)
mesh_1           | 	at app//io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163)
mesh_1           | 	at app//io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:510)
mesh_1           | 	at app//io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:388)
mesh_1           | 	at app//io.netty.util.concurrent.SingleThreadEventExecutor$6.run(SingleThreadEventExecutor.java:1044)
mesh_1           | 	at app//io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
mesh_1           | 	at app//io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
mesh_1           | 	at java.base@11.0.3/java.lang.Thread.run(Unknown Source)
```

**Comment by Jotschi** [comment]
It seems the schema generation takes longer than expected. We should only generate the example POJO's when needed and only once (not individually for each verticle).

**Comment by Jotschi** [comment]
Issue will be fixed via https://github.com/gentics/mesh/pull/997

**Comment by Jotschi** [comment]
Fix merged. Fix will be released with 1.4.0 - Thanks for the report
